server/src/ConfigManager.h

```diff
@@ -37,6 +37,7 @@ struct CommandLineOptions {
 	gboolean  disableCopyOnDemand;
 	gboolean  loadOldEvents;
 	gint      faceRestPort;
+	gint      faceRestWorkers;
```

Comment:
This variable doesn't have information about workers itself.
It just have the number of the workers.
How about using 'numFaceRestWorks'.